.travis.yml

```diff
@@ -12,6 +12,10 @@ cache:
     - $HOME/.composer/cache
     - repos/wordpress
 
+before_install:
+  - eval "$(ssh-agent -s)"
+  - echo -n "${base64_4950820e8b20_key}" | ssh-add -
```

Comment:
This would still need to be base64 decoded before adding, would it not?

Reply:
Oh yes.

Reply:
Done.